packages/confirm-dialog/src/vaadin-confirm-dialog.js

```diff
@@ -338,7 +341,13 @@ class ConfirmDialog extends ElementMixin(ThemePropertyMixin(ControllerMixin(Poly
       multiple: true,
       observe: false,
       initializer: (node) => {
-        this._messageNodes = [...this._messageNodes, node];
+        const wrapper = document.createElement('div');
```

Comment:
While the solution itself looks good, this use case of a wrapper node simply wasn't considered when I implemented the `SlotController` API in the first place. We probably should improve it somehow e.g. by supporting a template.

Not a blocker, just a possible future improvement idea (one of those I never have enough time to work on).